src/api/parser/src/index.js

```diff
@@ -0,0 +1,111 @@
+const { Satellite } = require('@senecacdot/satellite');
+const { router } = require('bull-board');
+// May need in next revision
+const parser = require('./routes/parser');
+const feedQueue = require('./feed/queue');
+const feedWorker = require('./feed/worker');
+const Feed = require('./data/feed');
+
+/**
+ * Adds the feed to the database if necessary, or gets a more complete
+ * version of the feed if we have better data already.
+ * @param {Object} feedData - feed data parsed from the wiki feed list.
+ * Returns Promise<Feed>, with the most appropriate Feed Object to use.
+ */
+async function updateFeed(feedData) {
```

Comment:
You're doing too much in this file.  You should split things up, so that this file just sets up the microservice with Satellite, and all this extra functionality can come from other modules.